File: client/Client/Network/Messages/Game/CreditPointsUpdateMessage.cs
namespace LoadRunnerClient.Network.Messages
{
    public class CreditPointsUpdateMessage
    {
        public const string TYPE = "CreditPointsUpdateMessage";

        public CreditPointsUpdateMessage()
        {
        }

        /// <summary>
        /// CreditPointsUpdateMessage Constructor
        /// </summary>
        /// <param name="uuid"></param>
        /// <param name="creditPoints"></param>
        public CreditPointsUpdateMessage(string uuid, int creditPoints)
        {
            this._uuid = uuid;
            this._creditPoints = creditPoints;
        }

        private string _uuid;

        public string uuid
        {
            get => _uuid;
            set => _uuid = value;
        }

        private int _creditPoints;

        public int creditPoints
        {
            get => _creditPoints;
            set => _creditPoints = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/game/CreditPointsUpdateMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.game;

public class CreditPointsUpdateMessage {

    public static final String TYPE = "CreditPointsUpdateMessage";

    private String uuid;
    private int creditPoints;

    public CreditPointsUpdateMessage() {
    }

    public CreditPointsUpdateMessage(String uuid, int creditPoints) {
        this.uuid = uuid;
        this.creditPoints = creditPoints;
    }

    public String getUuid() {
        return uuid;
    }

    public void setUuid(String uuid) {
        this.uuid = uuid;
    }

    public int getCreditPoints() {
        return creditPoints;
    }

    public void setCreditPoints(int creditPoints) {
        this.creditPoints = creditPoints;
    }
}
